packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePluginTest.kt

```diff
@@ -646,7 +694,6 @@ class AmplifyDataStorePluginTest {
 
         flutterPlugin.onSetUpObserve(mockResult)
 
-        verify(mockResult, times(1)).success(true)
```

Comment:
Can we update these with expected result? For example, `verify(mockResult, times(0))` or something